src/python/grpcio_tests/tests/unit/_contextvars_propagation_test.py

```diff
@@ -17,11 +17,13 @@
 import logging
 import os
 import sys
+import threading
 import unittest
 
 import grpc
 
 from tests.unit import test_common
+from six.moves import queue
```

Comment:
This is Python3 only. I guess we can just use `queue`.

Reply:
This functionality isn't actually Python 3 only. There are `contextvars` backports that can be run under 2.7 and my understanding is that such things are used in google3.